x-pack/plugins/observability_solution/profiling/public/plugin.tsx

```diff
@@ -64,29 +65,47 @@ export class ProfilingPlugin
     ];
 
     const kuerySubject = new BehaviorSubject<string>('');
+    const appUpdater$ = new BehaviorSubject<AppUpdater>(() => ({}));
 
     const section$ = combineLatest([from(coreSetup.getStartServices()), kuerySubject]).pipe(
       map(([[coreStart], kuery]) => {
         if (coreStart.application.capabilities.profiling.show) {
-          const sections: NavigationSection[] = [
-            {
-              label: i18n.translate('xpack.profiling.navigation.sectionLabel', {
-                defaultMessage: 'Universal Profiling',
-              }),
-              entries: links.map((link) => {
-                return {
-                  app: OBLT_PROFILING_APP_ID,
-                  label: link.title,
-                  path: `${link.path}?kuery=${kuery ?? ''}`,
-                  matchPath: (path) => {
-                    return path.startsWith(link.path);
-                  },
-                };
-              }),
-              sortKey: 700,
-            },
-          ];
-          return sections;
+          let isSidebarEnabled = true;
+          coreStart.chrome
+            .getChromeStyle$()
+            .pipe(take(1))
+            .subscribe((style) => (isSidebarEnabled = style === 'classic'));
+
+          if (isSidebarEnabled) {
+            // classic navigation
+            const sections: NavigationSection[] = [
+              {
+                label: i18n.translate('xpack.profiling.navigation.sectionLabel', {
+                  defaultMessage: 'Universal Profiling',
+                }),
+                entries: links.map((link) => {
+                  return {
+                    app: OBLT_PROFILING_APP_ID,
+                    label: link.title,
+                    path: `${link.path}?kuery=${kuery ?? ''}`,
```

Comment:
Q: Do we need the `kuery=''` in case the kuery is not defined - I would rather return the path alone so something like `kuery ? ${link.path}?kuery=${kuery} : link.path`, wdyt?

Reply:
Good point. I'll update conditions for both navigation.